gateway/webhook/webhook.go

```diff
@@ -313,7 +325,7 @@ func (bt *batchWebhookTransformerT) batchTransformLoop() {
 		bt.stats.sourceStats[breq.sourceType].numEvents.Count(len(payloadArr))
 
 		transformStart := time.Now()
-		batchResponse := bt.transform(payloadArr, breq.sourceType)
+		batchResponse := bt.transform(payloadWithSourceArr, payloadArr, breq.sourceType, "v1")
```

Comment:
We don't need to have version as argument here

Reply:
Done